jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/internal/HttpChannelState.java

```diff
@@ -1193,11 +1193,14 @@ public void write(boolean last, ByteBuffer content, Callback callback)
                 else
                 {
```

Comment:
Extra cleanup proposal:
`long totalWritten` and `HttpChannelState httpChannelState` could be moved within the `try` block.